code/datums/gamemode/factions/bloodcult/bloodcult_effects.dm

```diff
@@ -72,39 +72,47 @@
 	return
 
 
-/obj/effect/red_afterimage
+/obj/effect/afterimage
 	icon = null
 	icon_state = null
 	anchored = 1
 	mouse_opacity = 0
+	var/image_color
 
-/obj/effect/red_afterimage/New(var/turf/loc, var/atom/model)
+/obj/effect/afterimage/red
+	image_color = "red"
+
+/obj/effect/afterimage/New(var/turf/loc, var/atom/model, var/fadout = 5)
```

Comment:
not a big deal but why not fadeout instead?

Reply:
no good reason, it's a typo, just like me writing bellow instead of below all the time, but I've been improving.